apps/src/applab/applab.js

```diff
@@ -1747,9 +1747,11 @@ Applab.imageUploadButton = function (opts) {
 };
 
 // These offset are used to ensure that the turtle image is centered over
-// its x,y coordinates. The image is currently 31x45, so these offsets are 50%
-var TURTLE_X_OFFSET = -15;
-var TURTLE_Y_OFFSET = -22;
+// its x,y coordinates. The image is currently 44x44, with the center at
```

Comment:
an alternate image is available at 66x66, with the center at  (36,30), which could be rendered at a retina-friendly size of 33x33.